lib/order.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :ordered_dishes
+  PART_I_LENGTH = 42
+  PART_II_LENGTH = 8
+
+  def initialize
+    @ordered_dishes = []
+  end
+
+  def add_to_order(dish,quantity)
+    @ordered_dishes << {dish:dish,quantity:quantity}
+  end
+
+  def display_order
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for display_order is too high. [25.83/15]